data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -259,12 +259,23 @@ def _create_downloadable_data(
                     pd.to_numeric(df_100, errors="coerce")
                 ).astype("Int64")
                 df[column] = df_int
+
+            elif constants.FEMA_SUFFIX in column:
+                # Convert loss rates by multiplying by 100 (they are percents)
+                # and then rounding appropriately.
+                df_100 = df[column] * 100
+                df[column] = floor_series(
+                    series=df_100.astype(float64),
+                    number_of_decimals=constants.TILES_FEMA_ROUND_NUM_DECIMALS,
+                )
+
             else:
                 # Round all other floats.
                 df[column] = floor_series(
                     series=df[column].astype(float64),
                     number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
                 )
+                logger.info(df[column].dtype)
```

Comment:
Did you mean to remove this after using it for sanity-checking?

Reply:
yes, oof! sorry about that!